src/ng-backstretch.css

```diff
@@ -0,0 +1,8 @@
+.backstretch img {
+  opacity: 0;
+  transition: opacity 1s ease;
+}
+
+.backstretch img.active {
+  opacity: 1;
+}
```

Comment:
For the sake of keeping this sample code as simple as possible, there is not much CSS here and I think it best to just keep it all in the index.html file.